### PR TITLE
A first attempt at adding the Fermi-Dirac distribution

### PR DESCRIPTION
**Pull request by jeettrivedi**
Thank you for contributing to PlasmaPy!  Please check out our development guide at:

http://docs.plasmapy.org/en/master/development/index.html

Here are a few things to keep in mind when making a pull request (PR).

* Tests and docstrings are required for new or changed functionality.
* Please take care to write helpful commit messages: https://chris.beams.io/posts/git-commit/
* As a PR nears completion, fellow developers will likely suggest changes
  to be made before the PR is merged.
* Every pull request should include a changelog entry. Please see
  `changelog/README.rst` for instructions.
* PlasmaPy follows the PEP 8 style guide: https://www.python.org/dev/peps/pep-0008
* Please double check that a new PR will not duplicate an existing PR.
* If this PR will solve an issue tracked by GitHub, please add a phrase like
  "Closes #404." to automatically close the issue once the pull request is
  merged.  If your PR will not completely solve the issue, please still
  reference the issue.
* If this is your first contribution, please add your name to the author
  list in `docs/about/credits.rst`.
* Feel free to chat with other developers on our Matrix channel at:
  https://riot.im/app/#/room/#plasmapy:matrix.org

### CODE REVIEW

**Comment by jeettrivedi** [comment]
I will update my code in accordance with the PEP8 style guide and push the changes asap

File: plasmapy/physics/distribution.py [hunk 1]
```diff
@@ -16,17 +17,114 @@
...
+    ValueError
+        If the temperature is negative, or the particle mass or charge state
+        cannot be found.
+
+    Notes
+    -----
+
+    Examples
+    --------
+
```
Comment (StanczakDominik) [inline plasmapy/physics/distribution.py]: The docstring could really use a bit more of a description, a reference to the source for the formula, and some example docstrings! IPython's `%doctest_mode` is helpful for the latter`.

Comment (jeettrivedi) [reply plasmapy/physics/distribution.py]: Thank you, I will do improve the docstring and write unit tests. I will also implement a 2D and 3D version of it during the week

**Comment by StanczakDominik** [comment]
Hey, how are those docstrings and unit tests coming along? Is there way I could help you, @jeettrivedi? :)

**Comment by jeettrivedi** [comment]
> Hey, how are those docstrings and unit tests coming along? Is there way I could help you, @jeettrivedi? :)

Hey, I've been a bit caught up this past week, I intend to get them done this week along with the 2D and 3D implementations.

**Pull request closed without merging**